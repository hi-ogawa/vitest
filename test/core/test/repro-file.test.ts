import { it } from 'vitest';

it.concurrent('1st', ({ expect }) => {
  expect("hi").toMatchSnapshot();
});

it.concurrent('2nd', ({ expect }) => {
  expect("hi").toMatchSnapshot();
});
