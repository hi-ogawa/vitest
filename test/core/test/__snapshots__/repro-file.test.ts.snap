// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`2nd 1`] = `"hi"`;
